core/rust/src/api/http.rs

```
@@ -0,0 +1,640 @@
+        metrics.insert("status".to_string(), "healthy".to_string());
+        metrics.insert("version".to_string(), env!("CARGO_PKG_VERSION").to_string());
+        metrics
+    }
+}
+
+/// JSON representation of intent request
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct IntentRequestJson {
+    pub signals: Vec<SignalInputJson>,
+    pub client_id: String,
+    #[serde(default)]
```

Clippy (1.92.0)

[error] cannot find attribute `serde` in this scope

(error)